awx/ui/src/screens/Inventory/Inventory.js

```diff
@@ -167,6 +167,9 @@ function Inventory({ setBreadcrumb }) {
                     inventory.id,
                   or__workflowjob__inventory: inventory.id,
                 }}
+                additionalRelatedSearchableKeys={[
+                  'inventoryupdate__inventory_source__inventory',
```

Comment:
Adding this key here is displaying this one is part of the advanced search.

![image](https://user-images.githubusercontent.com/9053044/149789366-036c62e9-cd9e-4617-b2d8-5b573ca32df5.png)

I tested the search, and it seems to be working fine.